app/routes.js

```diff
@@ -6,7 +6,7 @@ import {
 } from "react-router";
 import App from "./components/App";
 import HomePage from "./components/home/HomePage";
-import UsersPage from "./components/users/UsersPage";
+import UsersPageComponent from "./components/users/UsersPage";
```

Comment:
Not sure why this change but I liked the previous one 😄.

Reply:
This is not only a naming thing. This was throwing a warning. Why? because UsersPage is a decorated component using redux. As the comment on the code test: 
`Using the undecorated component (In order to be able to test the component itself without having to deal with the decorator)`. 
So that the component have 2 exports, the default decorated and the undecorated without redux, only react. To be able to import correctly without warnings, we need to change the name of the imported component when using it (ie: routes). This is the same that we applied on VAS. Thanks

Reply:
@juampick I didn't know that! Thanks for sharing.

BTW, I redacted part of your comment. Remember this is a public repository and we cannot share the specifics of what we do for each of our clients.

Reply:
You are right. Thanks JD.